caffe2/proto/torch.proto

```diff
@@ -383,6 +421,17 @@ message TensorProto {
   // When this field is present, the data_type field MUST be
   // UINT32 or UINT64
   repeated uint64 uint64_data = 11 [packed = true];
+
```

Comment:
There's two additional things that we store for Parameters that don't have a place to go: is_buffer and requires_grad. Both are booleans.